### PR TITLE
Ra gh1254 cli changes

### PR DESCRIPTION
**Pull request by rhiananthony**
### Purpose
The purpose of this ticket is to detangle the coupling of the deployment of WFL with the WFL repo. 

This is the ticket for this work: https://broadinstitute.atlassian.net/browse/GH-1254

### Changes
Code related to deployment of the workflow launcher has been removed from this repo. This includes 

### Review Instructions
The remaining code should be necessary for either the 'release' or the 'tag-and-push-images' commands. Code not relevant to these two commands should be removed.

### CODE REVIEW

File: ops/cli.py [hunk 1]
```diff
@@ -230,58 +53,15 @@ def publish_docker_images(config: WflInstanceConfig) -> None:
...
-    """Print success and get pods from existing kubectl context."""
-    success(f"{config.command} is done!")
-    info(shell("kubectl get pods"), plain=True)
-
-
 def make_git_tag(config: WflInstanceConfig) -> None:
     info("=>  Tagging current commit with version")
     shell(f"git tag -a v{config.version} -m 'Created by cli.py {config.command}'", cwd=config.wfl_root_folder)
     shell(f"git push origin v{config.version}", cwd=config.wfl_root_folder)
     success(f"Tag 'v{config.version}' created and pushed")
```
Comment (ehigham) [inline ops/cli.py:60]: This can go

File: ops/cli.py [hunk 2]
```diff
@@ -58,170 +38,13 @@ def read_version(config: WflInstanceConfig) -> None:
...
-    info(f"To disconnect and stop the container, run: \n\t docker stop {config.cloud_sql_local_proxy_container}")
-
-
-def prompt_deploy_version(config: WflInstanceConfig) -> None:
-    """Verify that the user would like to deploy the stored version."""
-    if not input(f"Are you sure you want to deploy version {config.version}? [N/y]: ").lower().startswith("y"):
-        exit(0)
-
-
 def publish_docker_images(config: WflInstanceConfig) -> None:
```
Comment (ehigham) [inline ops/cli.py:48]: This can also go

**Review by ehigham — APPROVED** [review]
Thanks you - a few more functions to kill plz

File: ops/cli.py (hunk 1 above)
Comment (rhiananthony) [reply ops/cli.py]: Done

File: ops/cli.py (hunk 2 above)
Comment (rhiananthony) [reply ops/cli.py]: Done

**Pull request merged**